src/NewCard.js

```diff
@@ -1,31 +1,28 @@
 import React, { Component } from 'react'
 import Input from './Input'
 
-export default class FlashCard extends Component {
+export default class NewCard extends Component {
   constructor(props) {
     super(props)
-    this.state = { cards: [] }
     this.handleSubmit = this.handleSubmit.bind(this)
   }
 
   handleSubmit(event) {
     event.preventDefault()
-
+    
     const formData = new FormData(event.target)
     const newCard = {}
     for (let pair of formData.entries()) {
       newCard[pair[0]] = pair[1]
     }
-    const cards = [...this.state.cards, newCard]
+    const cards = [...this.props.currentCards, newCard]
```

Comment:
You form should not need to know about the entire cards list, just the one it's creating. You can pass the created card up to the `App`.